src/cript/nodes/uuid_base.py

```diff
@@ -32,16 +29,26 @@ class JsonAttributes(BaseNode.JsonAttributes):
 
     _json_attrs: JsonAttributes = JsonAttributes()
 
+    def __new__(cls, *args, **kwargs):
+        uuid: Optional[str] = kwargs.get("uuid")
```

Comment:
this fixes the bug that we had before.

Fun playing with something new like the `__new__`